framework/tests/all-systems/InferNullType.java

```diff
@@ -4,8 +4,13 @@
         return input;
     }
 
+    <T> T toInfer2(T input) {
```

Comment:
Please add examples with multiple type variables, e.g `<T extends @Nullable Number, S extends T>` and then try null in the different possible combinations.